package.json

```diff
@@ -17,9 +17,10 @@
   },
   "dependencies": {
     "@docusaurus/core": "3.8.1",
+    "@docusaurus/plugin-client-redirects": "^3.8.1",
```

Comment:
Please pin the version (`3.8.1` not `^3.8.1`), otherwise renovate will just raise a PR to pin this straight afterwards. Use `npm add --exact` in future to avoid this.